classes/submission/maps/Schema.php

```diff
@@ -283,12 +283,6 @@ protected function getPropertyReviewAssignments(Submission $submission): array
 
         $reviews = [];
         foreach ($reviewAssignments as $reviewAssignment) {
-            // @todo for now, only show reviews that haven't been
-            // declined or cancelled
-            if ($reviewAssignment->getDeclined() || $reviewAssignment->getCancelled()) {
-                continue;
-            }
-
```

Comment:
This will add declined and cancelled review assignments into the submission Schema. For now, we don't separate the schema data which goes to the submissions list and single submission calls. E.g., this also will be included into the response to the `...submission/{submissionId}` endpoint

Reply:
Can you think any unexpected consequence outside the submission summary I am working on?

Reply:
Sorry, the cancelled date was added by another commit. Nevertheless, I think exposing declined and cancelled reviews to authors and reviewers might be sensitive. What about showing those only for submissions list?

Reply:
@Vitaliy-1 I agree to exclude these (declined/cancelled) to authors.  Just not sure where to express this logic. This change was really just to be able to display all the review assignment for editors, but happy to remove it from this PR if you address it properly in some other PR - so we would have reduced version of review assignments for authors and reviewers.